test/sanity/sanity_test.go

```diff
@@ -65,13 +65,17 @@ func TestSanity(t *testing.T) {
 	enableStoragePools := false
 	multiZoneVolumeHandleConfig := driver.MultiZoneVolumeHandleConfig{}
 	listVolumesConfig := driver.ListVolumesConfig{}
+	provisionableDisksConfig := driver.ProvisionableDisksConfig{
+		SupportsIopsChange:       []string{"pd-standard"},
+		SupportsThroughputChange: []string{"pd-standard"},
```

Comment:
Why did you choose pd-standard for this?

Reply:
I chose pd-standard as there was an issue with hyperdisk creation causing listVolume tests to fail. I updated the logic for the ListVolumes to convert betadisks to disks